sources/packages/web/src/services/http/ConfirmationOfEnrollmentApi.ts

```diff
@@ -42,18 +42,20 @@ export class ConfirmationOfEnrollmentApi extends HttpBaseClient {
 
   public async confirmCOE(
     locationId: number,
-    applicationId: number,
+    disbursementScheduleId: number,
   ): Promise<void> {
-    try {
-      await this.apiClient.patch(
-        `institution/location/${locationId}/confirmation-of-enrollment/application/${applicationId}/confirm`,
+    await this.apiClient
+      .patch(
+        `institution/location/${locationId}/confirmation-of-enrollment/disbursement/${disbursementScheduleId}/confirm`,
         {},
         this.addAuthHeader(),
-      );
-    } catch (error) {
-      this.handleRequestError(error);
-      throw error;
-    }
+      )
+      .catch(error => {
```

Comment:
Please follow the try/catch format as the other methods.

Reply:
used patchCall.